tests/end2end/features/completion.feature

```diff
@@ -95,3 +95,9 @@ Feature: Using completion
         Then the following tabs should be open:
             - data/hello2.txt (active)
             - data/hello.txt
+
+    Scenario: Space updates completion model after selecting full command
+        When I run :set-cmd-text :set
+        And I run :completion-item-focus next
+        When I run :set-cmd-text -s :set
```

Comment:
This should be `And`, but you copied that from @rcorre :wink: I'll fix it up when merging.